docs/contributors/getting-started.md

```diff
@@ -31,54 +30,30 @@ To install Docker Compose, [follow their instructions here](https://docs.docker.
 Once Docker is installed and running, run this script to install WordPress, and build your local environment:
 
 ```
-npm run env install
+npx wp-env start
 ```
-
-#### Alternative Method: Using an Existing Local WordPress Install
```

Comment:
I removed the entire section since I wasn't how to fit this in the bigger picture anymore; I had to modify a lot of the commands in subsequent sections to use `wp-env`, which wouldn't work with the instructions in this section. Okay to retire the alternative method section?

Reply:
:+1:

Reply:
`wp-env` does support this with the config file PR. We might want to mention it here.

Reply:
Done in 743c26ca21f447b2889d3f1d07664c860fcc671d :)